### ISSUE TITLE
Joints label

### ISSUE THREAD

**Issue by CodingNovice7** [AutodeskAILab/Fusion360GalleryDataset#98]
In the assembly dataset, it seems that the Joints of the assembly are not complete, resulting in many parts not connected to any other parts from the Joints information, is there any way to complete the labels of these Joints?

**Comment by karldd** [comment]
Yes when designers create assemblies they may chose to only add joints in specific locations. We provide the data as designed, without additional annotations.

**Comment by CodingNovice7** [comment]
I have seen some assemblies in which a large number or even all parts are loaded with unit pose matrix and still assembled successfully, why is this the case?
